cfme/networks/views.py

```diff
@@ -735,7 +736,7 @@ def is_displayed(self):
         title = '{name} (All Security Groups)'.format(name=self.context['object'].name)
         return (super(BaseLoggedInPage, self).is_displayed and
                 self.navigation.currently_selected == ['Networks', 'Providers'] and
-                self.title.text == title)
+                self.entities.title.text == title)
```

Comment:
I don't think `title` widget exists on the entities view in this case. `SecurityGroupEntities` doesn't have it, and `BaseEntitiesView` doesn't have it.

The title widget definition on L729 above needs to be moved into the `SecurityGroupEntities` view class.

Reply:
No need of it. It was my bad. entities have a `title`

Reply:
alright, so the title widget that was added won't be needed.

Reply:
👍 I removed Mike. It was my bad I was unknown about `BaseEntityConditionalView` even not checked with MIQ shell.